hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-router/src/main/java/org/apache/hadoop/yarn/server/router/clientrm/DefaultClientRequestInterceptor.java

```diff
@@ -116,16 +120,15 @@ public void init(String userName) {
     final Configuration conf = this.getConf();
     try {
       clientRMProxy =
-          user.doAs(new PrivilegedExceptionAction<ApplicationClientProtocol>() {
-            @Override
-            public ApplicationClientProtocol run() throws Exception {
-              return ClientRMProxy.createRMProxy(conf,
-                  ApplicationClientProtocol.class);
-            }
-          });
+          user.doAs((PrivilegedExceptionAction<ApplicationClientProtocol>) () ->
```

Comment:
The readability of the same in DefaultRMAdminRequestInterceptor is better.
I would split the line with the doAs from the lambda.

Reply:
Thanks for your suggestion! I will modify the code.